package.json

```diff
@@ -17,7 +17,8 @@
     "test:watch": "yarn test --watch",
     "test:staged": "yarn test --bail --findRelatedTests --forceExit",
     "test:ci": "yarn test --coverage",
-    "build": "rimraf ./dist && sucrase ./packages -d ./dist --transforms typescript,imports"
+    "build": "rimraf ./dist && sucrase ./packages -d ./dist --transforms typescript,imports",
+    "build:lerna": "rimraf ./packages/**/dist && lerna run build"
```

Comment:
Why do two script of build?

Reply:
Beacuse command `build: lerna` will run command build in all pacakges (according to yours configuration) and create the distribution in speficied folder where each command build was runned, already the command `build` in global scope it transpile the packages in root folder of distribution.